transforms/ember-object/__testfixtures__/options/decorators-disabled.input.js

```diff
@@ -4,74 +4,74 @@ import { inject as service } from "@ember/service";
 import { on } from "@ember/object/evented";
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo1 = EmberObject.extend({
   computed: computed({
     get() {},
     set() {}
   })
 });
 
 // Do not transform
-const Foo = EmberObject.extend({
+const Foo2 = EmberObject.extend({
```

Comment:
More generally, it's weird that we don't transform these into:

```ts
class Foo2 extends EmberObject.extend({
  statefulObject: {},
  statefulArray: []
}) {}
```

Reply:
IIRC these are a linter error anyway indicating one should use `Object.freeze` or similar. Probably should suggest fixing non-Octane ember linter errors before running this codemod.